source/reference/options.txt

```diff
@@ -419,11 +419,12 @@ Authentication Options
    If :option:`--sspiHostnameCanonicalization 
    <--sspiHostnameCanonicalization>` is set to:
 
-   - ``forward``, the effect is the same as setting ``authMechanismProperties=CANONICALIZE_HOST_NAME:true``.    
-   - ``none``, the effect is the same as setting ``authMechanismProperties=CANONICALIZE_HOST_NAME:false``.    
-
-   :option:`--sspiHostnameCanonicalization 
-   <--sspiHostnameCanonicalization>` cannot be set to ``forwardAndReverse``.  
+   - ``forwardAndReverse``, ``mongosh`` 1.3.0 adds support to perform a
```

Comment:
[suggestion]

Consider breaking this into two clauses.

Spitballing:

- ``forwardAndReverse``, forward DNS lookup and reverse DNS lookup are supported. Added in ``mongosh`` 1.3.0.

Reply:
Both are supported, but the important thing is that it does both during the lookup.

Updated

Reply:
Got it! 👍